### DISCUSSION TITLE
Docker Kerberos authentification windows

### DISCUSSION THREAD

**Discussion by cricrack76** [semaphoreui/semaphore#2572] [Show and tell]
Hi,
I think my request relates to the following problem : https://github.com/semaphoreui/semaphore/discussions/2490

When I wanted to use your docker image, I couldn't connect to my windows machines via Kerberos/AD authentication.
I had an error when running docker because it failed to add the krb5, kerberos and gssapi modules

I modified your image via this dockerfile.
--
FROM semaphoreui/semaphore:v2.10.35
USER root
RUN apk add --no-cache -U --upgrade krb5-libs krb5 libressl-dev cyrus-sasl-gssapiv2 krb5 krb5-dev gcc musl-dev python3-dev
COPY krb5.conf /etc/krb5.conf
--
and in the requirements.txt :
# for common jinja-filters
netaddr
jmespath
# for common modules
pywinrm
passlib
requests
docker
pyvmomi
gssapi
krb5
kerberos

--
After these changes I was able to connect to my windows servers via kerberos/AD.

Can you add the necessary in your docker image to allow this connection?
Otherwise can you indicate it in your documentation?

Thanks

**Comment by fiftin** [comment]
Could you create PR?

**Comment by cricrack76** [comment]
Thank you for your response.
I've never done a PR before but I tried.
Hopefully I made the changes in the right place.
